caffeinated/admin/new/pricing/templates/event_tickets_datetime_dtt_tickets_list.template.php

```diff
@@ -12,12 +12,12 @@
  */
 ?>
 
-<li data-datetime-row="<?php echo $dtt_row; ?>" data-context="datetime-ticket" data-ticket-row="<?php echo $tkt_row; ?>"
+<li data-datetime-row="<?php echo esc_attr($dtt_row); ?>" data-context="datetime-ticket" data-ticket-row="<?php echo esc_attr($tkt_row); ?>"
```

Comment:
OOF!

the phpdocs for this template show:

```php
 * @var string $dtt_row
 * @var string $tkt_row
```

but those are surely integers, so can use `absint()` instead of `esc_attr()`